decidim/decidim
# Show only ToS acceptance when admin hasn't accepted it

#### :tophat: What? Why?

On the last two weeks I heard two times this same bug report so I went ahead and tried to tackle it. 

![image](https://user-images.githubusercontent.com/717367/196207071-17ddde75-c4a2-46aa-b573-48c98ad2bdb3.png)

Basically, the problem is that there are some elements that distract the admin when logging in, such as the "Activity" (user statistics), and the admin main menu (Initiatives, Global Moderations, Participants and Templates). 


#### :pushpin: Related Issues

- Related to #5507

#### Testing

1. Reject the admin terms, for instance by running this in a console: `Decidim::User.first.update(admin_terms_accepted_at: nil)   `
3. Sign in as this account (admin@example.org with the defaults seed) 
4. Go to /admin
5. See the acceptance terms screen

### :camera: Screenshots

#### Before

![image](https://user-images.githubusercontent.com/717367/196207377-8d80d6b9-dc3a-45d2-8d94-5402aefa82a5.png)


#### After

![image](https://user-images.githubusercontent.com/717367/196941179-3d2adf22-3fa2-4ac2-bb31-2cd0c226fac7.png)



:hearts: Thank you!
